src/codeu/chat/server/Controller.java

```diff
@@ -189,6 +189,32 @@ public User newUser(Uuid id, String name, Time creationTime) {
     return user;
   }
 
+
+  @Override
+  public void addMember(Uuid user, Uuid conversation) {
+    ifIdFree(user) {
```

Comment:
id can also belong to conversation and other objects.

better check against dedicated user store